src/styles.css

```diff
@@ -0,0 +1,34 @@
+body {
+  color: white;
+  background-image: url("./assets/background.jpg");
```

Comment:
- [ ] This background should be changing according to the response/weather in the submitted city.
You can change it simply by creating classes for different weathers and toggling them according to the API response.